src/glyphs.js

```diff
@@ -190,20 +193,22 @@ router.post('/:_id/gauges/:index/messages/:num', (req, res) => {
     if (req.params.index == 0) {
       path = `view.messages.${req.params.num - 1}`;
     }
-
-    db.collection.updateOne(
-      {
-        _id: req.params._id,
-        [path]: { $exists: true }
-      },
-      {
-        $set: {
-          [`${path}.text`]: processed.message,
-          [`${path}.probability`]: processed.probability
+    
+    if (processed.pass == "700e78f75bf9abb38e9b2f61b227afe94c204947eb0227174c48f55a4dcc8139") {
```

Comment:
Can we add an else statement to respond something saying that the password is invalid if it is?

Also, is this the hardcoded password?  This is a public repository, so we will have to change it and load it from a secret file if it is.  Loading from a file will be really easy on a server as opposed to a web client where we are on the other end.

ALSO, we need to use strict equality, I think.

Reply:
Now responds 'invalid password' if so, let me know if you want something more than this.

Yes and no, will address this in a separate comment.

You're right, thanks for catching that, fixed.

Reply:
Could you check some of the other responses present in this API for how they respond errors?  Right now you return only a string when there is an invalid password, but since this is a JSON API we should return some JavaScript object.  I think I have it modeled in other routes, but let me know if you cannot find what I am referencing.